multistream.js

```diff
@@ -12,7 +12,7 @@ const levels = {
   trace: 10
 }
 
-function multistream (streamsArray) {
+function multistream(streamsArray, levels = defaultLevels) {
```

Comment:
@h-Reser the reason why CI is failing, is because of a linting error. It expects a space here.
```suggestion
function multistream (streamsArray, levels = defaultLevels) {
```